controllers/deactivation/deactivation_controller.go

```diff
@@ -56,9 +54,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	logger := log.FromContext(ctx)
 	logger.Info("Reconciling Deactivation")
 
-	config, err := toolchainconfig.GetConfig(r.Client, request.Namespace)
+	config, err := toolchainconfig.GetToolchainConfig(r.Client)
```

Comment:
I have one more question - why do you use `GetToolchainConfig` and not `GetCachedToolchainConfig`?

Reply:
I created `GetCachedToolchainConfig` for use in the registration service because there isn't always a client available in all the places we wanted to get config values. So the simple reason is I used `GetToolchainConfig` because I started the host operator migration before I did the reg service changes. 🙂 

Since the toolchainconfig controller updates the cache it's probably fine for us to switch to `GetCachedToolchainConfig` but I think it's okay to keep using `GetToolchainConfig` since it can handle the edge case where config is used before the toolchainconfig controller has initialized the cache.

Reply:
As I see it - we should ideally use only the `GetToolchainConfig` function. 
Maybe we could remove the `GetCachedToolchainConfig` function from the code to avoid confusion. If I'm not mistaken, it's used only in tests now.